src/components/pages/404-pages/Wallpaper.vue

```diff
@@ -0,0 +1,35 @@
+    </svg>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    color: String,
+  },
+}
+</script>
+
+<style lang="scss">
+.page-not-found__wallpaper {
```

Comment:
Might make sense to do positioning in parent and keep this svg being svg?
Up to you.